docs/android-mobile-gestures.md

```diff
@@ -105,6 +105,10 @@ driver.executeScript("mobile: clickGesture", ImmutableMap.of(
 driver.execute_script('mobile: clickGesture', {'x': 100, 'y': 100})
 ```
 
+```javascript
+// Javascript - Mocha
```

Comment:
Mocha is a general-purpose test runner framework. I meant the javascript wrapper over Appium, like WebdriverIO.

Reply:
You are right, I'm using wdio, so is it ok if I change the line to :

`// Javascript - @wdio
`

or to write it somehow differently?

Reply:
yes, it works